script.js

```diff
@@ -1,4 +1,146 @@
+var attempts = 0;
+var wins = 0;
+var playerName = "";
+var gamemode = "normal";
+var currentWinner = "";
+
+// right now attempts & win count carries over for normal and reversed game, but not korean version
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  if (playerName == "") {
+    playerName = input;
```

Comment:
I like how you intro the user to the game, well done!
Another possibility would have been to directly call
`playerName = input;` if that was what you were trying to achieve